Documentation/network/egress-gateway.rst

```diff
@@ -205,6 +205,20 @@ One or more IPv4 destination CIDRs can be specified with ``destinationCIDRs``:
     pods, nodes, Kubernetes API server) will be excluded from the egress gateway
     SNAT logic.
 
+It's possible to specify exceptions to the ``destinationCIDRs`` list with
+``excludedCIDRs``:
+
+.. code-block:: yaml
+
+    destinationCIDRs:
+    - "a.b.0.0/16"
+    excludedCIDRs:
+    - "a.b.c.0/24"
+
+In this case traffic destined to the ``a.b.0.0/16`` CIDR, except for the
+``a.b.c.0/24`` destination, will go through egress gateway and leave the cluster
```

Comment:
Maybe we should make the two CIDRs completely unrelated in the example to not give the wrong impression that the excludedCIDR needs to be a subset of the destinationCIDR?

Reply:
Wrong impression? My understanding was the opposite?

```
              excludedCIDRs:
                description: ExcludedCIDRs is a list of destination CIDRs that will
                  be excluded from the egress gateway redirection and SNAT logic.
                  Should be a subset of destinationCIDRs otherwise it will not have
                  any effect.
```

Reply:
I think I was confused on how CIDRs work :sweat_smile: If two CIDRs overlap, then they are either equal or one is the subset of the other.